aicoe/add-image-to-datahub.md

```diff
@@ -1,10 +1,11 @@
 # Import Container Image to Internal Datahub:
 
-1. Create a new ImageStream template, you can use [this template](https://github.com/AICoE/s2i-custom-notebook#how-to-add-the-image-to-open-data-hubjupyterhub).
-2. Fork this repo https://gitlab.cee.redhat.com/data-hub/dh-jupyterhub.
-3. Add a new ImageStream to the [bases/custom-images/jupyterhub-custom-images.yaml](https://gitlab.cee.redhat.com/data-hub/dh-jupyterhub/blob/master/bases/custom-images/jupyterhub-custom-images.yaml).
-4. And create a Merge Request to publish these changes to the internal DataHub instance.
+* Create a new ImageStream template, you can use [this template](https://github.com/AICoE/s2i-custom-notebook#how-to-add-the-image-to-open-data-hubjupyterhub).
+* Fork this [repository](https://github.com/AICoE/aicoe-sre/tree/master/applications/jupyterhub/bases).
+* Add a new ImageStream to the [repository](https://github.com/AICoE/aicoe-sre/blob/master/applications/jupyterhub/bases/custom-images).
```

Comment:
the link should be https://github.com/AICoE/aicoe-sre/blob/master/applications/jupyterhub/bases/custom-images/jupyterhub-custom-images.yaml